### PR TITLE
Convert project to use AndroidX

### PR DESCRIPTION
**Pull request by tiembo**
Addresses #2

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,20 +24,20 @@ buildscript {
 
         // App dependencies
         constraintLayoutVersion = '2.0.0-alpha2'
-        espressoVersion = '3.0.1'
+        espressoVersion = '3.1.0-alpha4'
         glideVersion = '4.4.0'
         gradleVersion = '3.2.0-rc03'
         gsonVersion = '2.8.2'
         junitVersion = '4.12'
         kotlinVersion = '1.2.70'
```
[the remark this answers is not in the document]
Comment (tiembo) [reply build.gradle]: I agree.  Sunflower uses many alpha, beta, and RC libs so we should move to Kotlin 1.3.  That's outside of the scope of this PR though - would you want to open a new PR for it?

[the remark this answers is not in the document]
Comment (XinyueZ) [reply build.gradle]: no problem, I will ship one PR for `kotlin 1.3 m2` to keep this PR focusing on support2androix only.

File: app/build.gradle [hunk 2]
```diff
@@ -47,27 +47,28 @@ android {
...
+    implementation "com.google.android.material:material:$rootProject.supportLibraryVersion"
     implementation "com.google.code.gson:gson:$rootProject.gsonVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$rootProject.kotlinVersion"
 
     // Testing dependencies
-    androidTestImplementation "com.android.support.test.espresso:espresso-intents:$rootProject.espressoVersion"
-    androidTestImplementation "com.android.support.test.espresso:espresso-contrib:$rootProject.espressoVersion"
-    androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
-    androidTestImplementation "com.android.support.test.uiautomator:uiautomator-v18:$uiAutomatorVersion"
+    androidTestImplementation "androidx.arch.core:core-testing:2.0.0-rc01"
```
Comment (thagikura) [inline app/build.gradle]: Define a variable to be consistent?

**Comment by tiembo** [comment]
Thanks @thagikura!

**Pull request merged**